src/history/test/HistoryTestsUtils.cpp

```diff
@@ -766,9 +758,42 @@ CatchupSimulation::catchupOnline(Application::pointer app, uint32_t initLedger,
     }
 
     uint32_t triggerLedger = hm.ledgerToTriggerCatchup(firstLedgerInCheckpoint);
-    for (uint32_t n = initLedger; n <= triggerLedger + bufferLedgers; ++n)
+
+    // this option will change the ordering of the triggerLedger and ledgers
+    // after it
+    if (randomizeBufferedLedgers)
+    {
+        for (uint32_t n = initLedger; n < triggerLedger; ++n)
+        {
+            externalize(n);
+        }
+
+        std::vector<uint32_t> bufferedLedgers(bufferLedgers);
+        std::iota(bufferedLedgers.begin(), bufferedLedgers.end(),
```

Comment:
`std::iota` doesn't build for me (missing include?)

Reply:
removed iota